cluster/gce/config-common.sh

```diff
@@ -159,3 +159,7 @@ export WINDOWS_BOOTSTRAP_KUBECONFIG_FILE="${WINDOWS_K8S_DIR}\kubelet.bootstrap-k
 export WINDOWS_KUBEPROXY_KUBECONFIG_FILE="${WINDOWS_K8S_DIR}\kubeproxy.kubeconfig"
 # Pause container image for Windows container.
 export WINDOWS_INFRA_CONTAINER="gcr.io/gke-release/pause-win:1.2.1"
+# Storage Path for csi-proxy
+export CSI_PROXY_STORAGE_PATH="https://storage.googleapis.com/gke-release-staging/csi-proxy"
+# Version for csi-proxy
+export CSI_PROXY_VERSION="v0.1.0-gke.1"
```

Comment:
Will these variables ever be used for non-windows platforms?

Reply:
no, csi-proxy is only for windows.

Reply:
Could you add that to the comment?

Reply:
you mean windows tests. We can trigger it here with /test pull-kubernetes-e2e-windows-gce?

Reply:
added.